### PR TITLE
fix: simulated fee estimations

### LINKED ISSUE #2667: Ensure that fee ceiling is applied upon first load of transaction signing screen
I've noticed that the default recommended Standard gas fee is (incorrectly?) much higher now on every transaction since version 3.17.0, around 0.2-0.5 STX or so typically. But if I open the Fees menu and press Standard again it goes back to the previous usual recommended fee of 0.003 STX. 

Is this an intended change or a bug?

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/3250446325).

This PR fixes the simulated fees loading in the UI before the capped api fee estimations.

### CODE REVIEW

File: tests/page-objects/wallet.page.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class WalletPage {
   }
 
   async waitForHomePage() {
-    await this.page.waitForSelector(this.$homePageContainer, { timeout: 30000 });
+    await this.page.waitForSelector(this.$homePageContainer, { timeout: 60000 });
```
Comment (kyranjamie) [inline tests/page-objects/wallet.page.ts]: Is this necessary?

File: src/app/query/stacks/fees/fees.hooks.ts [hunk 2]
```diff
@@ -61,7 +61,12 @@ export function useFeeEstimations(txByteLength: number | null, txPayload: string
     if (!isLoading && isError) {
       return { estimates: defaultFeeEstimations, calculation: FeeCalculationType.Default };
     }
-    if (!isLoading && (!!txFeeEstimation?.error || !feeEstimations?.length) && txByteLength) {
+    if (
+      txFeeEstimation &&
+      !isLoading &&
+      (!!txFeeEstimation?.error || !feeEstimations?.length) &&
+      txByteLength
+    ) {
```
Comment (kyranjamie) [inline src/app/query/stacks/fees/fees.hooks.ts]: This is quite a difficult to read expression. Could it be given a readable function name?

**Review by kyranjamie — APPROVED** [review]

File: tests/page-objects/wallet.page.ts (hunk 1 above)
Comment (fbwoolf) [reply tests/page-objects/wallet.page.ts]: I had to add this or the balance test was failing during CI waiting for the home page. Did your changes to the password validation cause any new delays? I can try to run them again today w/out this change, but just couldn't get it to pass yesterday. Reran them 3-4 times.

File: src/app/query/stacks/fees/fees.hooks.ts (hunk 2 above)
Comment (fbwoolf) [reply src/app/query/stacks/fees/fees.hooks.ts]: Sure, I can, I am trying to remember why this condition would happen? 🤔

**Pull request merged**